packages/components/src/TabBar/TabBar.scss

```diff
@@ -10,13 +10,62 @@
 	background-image: url('data:image/svg+xml;utf8,<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 16 16"><path d="M8 13.6l-8-8L2.6 3 8 8.4 13.4 3 16 5.6z" fill="#{hex-to-url($color)}"/></svg>');
 }
 
+$underline-height: 2px;
+
 .tc-tab-bar {
 	white-space: nowrap;
 	overflow: hidden;
 
 	& &-item {
 		float: none;
 		display: inline-block;
+
+		&-icon {
+			margin-right: $padding-smaller;
+			width: $svg-sm-size;
+			height: $svg-sm-size;
+			font-size: $svg-sm-size;
+		}
+
+		button {
+			padding: $padding-smaller $padding-small;
+			padding-bottom: calc(#{$padding-smaller} + #{$underline-height});
+			color: $black;
+
+			max-width: 200px;
+			white-space: nowrap;
+			overflow: hidden;
+			text-overflow: ellipsis;
+
+			&:hover {
+				color: $lochmara;
+			}
+
+			&[aria-selected='true'] {
+				position: relative;
+
+				font-size: 14px;
+				font-weight: semi-bold;
+				color: $lochmara;
+				box-shadow: none;
+
+				&:after {
+					position: absolute;
+					content: '';
+
+					left: $padding-small;
+					right: $padding-small;
+					bottom: 0;
+
+					border-bottom: solid $underline-height $lochmara;
```

Comment:
`$lochmara` is used 3 times, use variable instead like `$tc-tabbar-active-color`

Reply:
But $lochmara is already a global variable, why we need to reasign it to another local one?

Reply:
Actually `$lockmara` is a generated name for a specific color, it does not mean that it designed for active color